addons/binding/org.openhab.binding.pioneeravr/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: PioneerAvr Binding
+Bundle-SymbolicName: org.openhab.binding.pioneeravr;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: com.google.common.collect,
+ gnu.io;resolution:=optional,
+ org.apache.commons.lang,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.items,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.types,
+ org.jupnp.model.meta,
+ org.jupnp.model.types,
+ org.osgi.service.cm,
+ org.osgi.service.component,
+ org.slf4j
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.openhab.binding.pioneeravr,
+ org.openhab.binding.pioneeravr.discovery,

```

Comment:
Not sure, why I did not comment on this before, sorry: Is there any reason why  discovery and protocol packages are exported? If not, I'd suggest to rather move them to an internal package.